layout/index.jade

```diff
@@ -2,7 +2,13 @@ extends base
 - var current_title = page.title || tag.name || category.name || site.title
 
 block title
-  title= config.title + " | " + config.subtitle
+  if config.title
```

Comment:
I think nested if statements is bad. would you please change the code to utilize `? :` operator?

For example:
```
- var title = config.title ? config.title : "";
- title += config.subtitle ? " | " + config.subtitle : "";
```